api/tests/acceptance/application/cnav/auth-url-route-get_test.js

```diff
@@ -0,0 +1,41 @@
+const { expect } = require('../../../test-helper');
+const createServer = require('../../../../server');
+
+describe('Acceptance | Route | cnav auth-url', function () {
+  let server;
+
+  beforeEach(async function () {
+    server = await createServer();
+  });
+
+  describe('GET /api/cnav/auth-url', function () {
+    context('When the request returns 200', function () {
+      it('should return the url of authentication', async function () {
+        // when
+        const response = await server.inject({
+          method: 'GET',
+          url: `/api/cnav/auth-url?redirect_uri=${encodeURIComponent('http://app.pix.fr/connexion-cnav')}`,
+        });
+
+        // then
+        function _checkIfValidUUID(str) {
+          const regexExp = /^[0-9a-fA-F]{8}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{12}$/gi;
+          return regexExp.test(str);
+        }
+
+        expect(response.statusCode).to.equal(200);
+        expect(_checkIfValidUUID(response.result.state)).to.be.true;
+        expect(_checkIfValidUUID(response.result.nonce)).to.be.true;
```

Comment:
On pourrait être plus explicite :

```suggestion
        expect(response.statusCode).to.equal(200);
        const uuidRegex = /^[0-9a-fA-F]{8}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{12}$/gi;
        expect(response.result.state)).to.match(uuidRegex);
        expect(response.result.nonce)).to.match(uuidRegex);
```

Reply:
j'ai essayé mais la suggestion ne passe pas en l'état